src/App.js

```diff
@@ -1,25 +1,8 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './component/calculator';
 
 function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+  return (<Calculator />);
```

Comment:
- [ ] Good job so far :+1: Kindly make your `App` component a class-based component as stated in the requirements :100: